src/Routes/Devices/DeviceTable.js

```diff
@@ -170,7 +170,9 @@ const createRows = (devices, hasLinks) => {
 
 const DeviceTable = ({
   hasCheckbox = false,
+  //checkedRows,
```

Comment:
would be great to remove this commented code

Reply:
thank you! just removed that